src/audit/tide/tide.go

```diff
@@ -10,18 +10,17 @@ import (
 	"fmt"
 	"strings"
 	"reflect"
-	"github.com/xwp/go-tide/src/env"
```

Comment:
Only removing this one line seems off. Wouldn't we just import the new repo, or each package in the repo as needed?
```
https://github.com/wptide/pkg
```
OR
```
https://github.com/wptide/pkg/env
https://github.com/wptide/pkg/log
```

Reply:
Yeah scrap this whole PR.... its fixed in #9 by replacing all these imports completely.

Aside, Just removing that line is correct. We are no longer using `env.GetEnv` its passed as an optional value in the `result` object. If nothing is found, value will be empty (even for fallback) in which case the authenticated user will be used. (which is likely audit_server anyway)

But ignore all this, I'm closing the PR.

Final aside, trust the tests ;)